controller/index.js

```diff
@@ -1,22 +1,51 @@
 
 var pkg = require('../package');
+var markdown = require('markdown').markdown;
+var fs = require('fs');
 
 function setup(){
 
-  function controller( req, res, next ){
-
-    // stats
-    res.json({
-      name: pkg.name,
-      version: {
-        number: pkg.version
+  var text = null;
+  function getText( callback ) {
+    if( text ) {
+      process.nextTick( callback.bind( null, null, text ) );
+      return;
+    }
+
+    fs.readFile( './DOCS.md', 'utf8', function ( err, content ) {
```

Comment:
Can we use `fs.readFileSync()` instead? It'll only happen once so blocking the process is a non-issue, and would do away with the callback.